index.html

```diff
@@ -58,6 +58,23 @@
   </head>
 
   <body>
+    <button id="scroll-to-top-btn">
+      <svg
+      width="18"
+      height="11"
+      viewBox="0 0 18 11"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M2 9L9 2L16 9"
+        stroke="white"
+        stroke-width="4"
+        stroke-linecap="round"
+        stroke-linejoin="round"
+      />
```

Comment:
Can we please move this to image folder and use a Image tag here ??

Reply:
Image moved to the **img** folder and used an **<img>** tag instead